starlark/library.go

```diff
@@ -309,16 +309,28 @@ func unpackOneArg(v Value, ptr interface{}) error {
 	default:
 		ptrv := reflect.ValueOf(ptr)
 		if ptrv.Kind() != reflect.Ptr {
-			log.Fatalf("internal error: not a pointer: %T", ptr)
+			log.Panicf("internal error: not a pointer: %T", ptr)
 		}
 		param := ptrv.Elem()
 		if !reflect.TypeOf(v).AssignableTo(param.Type()) {
 			// Detect mistakes by caller.
+			log.Println(param.Type())
```

Comment:
Oops, the two log.Prints were relics of debugging.

I've clarified the comments and variable names, and called the Type() method using reflection instead of handling interfaces specially. PTAL.

Reply:
Thanks! This is much more readable.